src/puzzles/input.rs

```diff
@@ -0,0 +1,61 @@
+use std::fs::{self, File};
+use std::io::{self, BufRead};
+
+/// Reads input file into a single String.
+///
+/// This reads the file at "src/inputs/day_[day].txt" which should be
+/// downloaded from "https://adventofcode.com/2020/day/[day]/input".
+pub fn read_input(day: i32) -> String {
```

Comment:
Consider renaming there functions to not include the word `input`, as they will most likely get used in the context of `input::read_input`. `input::read` would get rid of the duplicate `input`.